ecommerce/mail_api.py

```diff
@@ -200,6 +200,9 @@ def send_course_run_enrollment_welcome_email(enrollment):
     Args:
         enrollment (CourseRunEnrollment): the enrollment for which to send the welcome email
     """
+    if not settings.FEATURES.get("ENROLLMENT_WELCOME_EMAIL", False):
```

Comment:
The feature flags are more readable when applied to the initiation of something. In this case, the flag should be placed where we are calling `send_course_run_enrollment_welcome_email` from. Someone looking at the code in api.py might think that the email would be sent but that would be blocked in this function. So, If we take this flag to the api.py before calling this function that would make more sense for code reader.

Reply:
It’s a good suggestion, but we will need to make this change in both places. Actually, I tried to make this change in a common place as it is being called not only through api.py but also through `courses/management/utils.py` for transfer enrollment.

Reply:
Alright, Let's keep it here then.

Reply:
> Alright, Let's keep it here then.

Sorry I was looking into addressing your request, and I pushed commit as didn't notice we were agreed upon.
I have reverted it. Now It seems we are good and ready to go to merge.